fetch.bs

```diff
@@ -4057,15 +4058,16 @@ optional <i>CORS-preflight flag</i>, run these steps:
   </ol>
 
  <li>
-  <p>If <var>request</var>'s <a for=request>response tainting</a> is "<code>opaque</code>"
-  and the <a>cross-origin resource policy check</a> with <var>request</var>'s
-  <a for=request>origin</a>, <var>request</var>'s <a for=request>client</a>, and
-  <var>actualResponse</var> returns <b>blocked</b>, then return a <a>network error</a>.
+  <p>If either <var>request</var>'s <a for=request>response tainting</a> or <var>response</var>'s
```

Comment:
I changed here because the previous logic didn't work well for responses coming from the service worker.